### PR TITLE
feat: membership badge in calls (AR-1940)

### PR DESCRIPTION
**Pull request by Garzas**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1940" title="AR-1940" target="_blank">StoryAR-1940</a>  [Part 2] FOLLOW UP TASKS for Display user metadata (membership qualifier) in Calling UI
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Display membership badge in incoming and outgoing calls

### Issues

_Briefly describe the issue you have solved or implemented with this pull request. If the PR contains multiple issues, use a bullet list._

### Causes (Optional)

_Briefly describe the causes behind the issues. This could be helpful to understand the adopted solutions behind some nasty bugs or complex issues._

### Solutions

_Briefly describe the solutions you have implemented for the issues explained above._

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-769/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-769/3/) **failed**.

File: app/src/main/kotlin/com/wire/android/ui/calling/ParticipantTile.kt [hunk 1]
```diff
@@ -143,6 +144,14 @@ fun ParticipantTile(
 
 @Preview
 @Composable
-@Suppress("EmptyFunctionBlock")
 private fun ParticipantTilePreview() {
+    ParticipantTile(
+        modifier = Modifier.height(300.dp),
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/calling/ParticipantTile.kt:149]: I believe we can use the centralized dimensions here :D

File: app/src/main/kotlin/com/wire/android/ui/common/MembershipQualifierLabel.kt [hunk 2]
```diff
@@ -11,30 +13,45 @@ import androidx.compose.ui.Modifier
...
-                .background(
-                    color = MaterialTheme.colorScheme.primaryContainer,
-                    shape = RoundedCornerShape(MaterialTheme.wireDimensions.corner4x)
-                )
-                .padding(horizontal = MaterialTheme.wireDimensions.spacing4x, vertical = MaterialTheme.wireDimensions.spacing2x)
-        )
+        Box(
+            modifier = Modifier.border(
+                width = 1.dp,
+                shape = RoundedCornerShape(4.dp),
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/common/MembershipQualifierLabel.kt:31]: I think we can use dimensions here as well

**Review by yamilmedina — APPROVED** [review]
small comments, nothing blocker, great work as usual 👏

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-769/4/) **failed**.

File: app/src/main/kotlin/com/wire/android/ui/calling/ParticipantTile.kt (hunk 1 above)
Comment (Garzas) [reply app/src/main/kotlin/com/wire/android/ui/calling/ParticipantTile.kt:149]: This is dynamically generated value, which depends on Phone screen 😅 So I just added random value for preview visualisation 😄

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-769/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3892331)(3892331)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3892331)(3892331)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/calling/ParticipantTile.kt (hunk 1 above)
Comment (yamilmedina) [reply app/src/main/kotlin/com/wire/android/ui/calling/ParticipantTile.kt:149]: Oh sorry didn't catch it was the preview block 😅

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-769/7/) **failed**.

**Comment by AndroidBob** [comment]
Build [8](https://10.10.124.134/job/wire-android-reloaded/job/PR-769/8/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3925627)(3925627)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3925627)(3925627)-dev-release.apk)

**Pull request merged**